liquidator/src/liquidator.js

```diff
@@ -4,22 +4,33 @@ const {
   revertWrapper
 } = require("@umaprotocol/common");
 
+const { LIQUIDATOR_CONFIG } = require("./settings");
+const { getLogger } = require("./common");
+
 class Liquidator {
   /**
    * @notice Constructs new Liquidator bot.
-   * @param {Object} logger Module used to send logs.
-   * @param {Object} expiringMultiPartyClient Module used to query EMP information on-chain.
-   * @param {Object} gasEstimator Module used to estimate optimal gas price with which to send txns.
-   * @param {Object} votingContract DVM to query price requests.
-   * @param {Object} priceFeed Module used to query the current token price.
-   * @param {String} account Ethereum account from which to send txns.
-   * @param {Object} empProps Contains EMP contract state data. Expected:
+   * @param {Object} param - Constructor params
+   * @param {Object} param.expiringMultiPartyClient Module used to query EMP information on-chain.
+   * @param {Object} param.gasEstimator Module used to estimate optimal gas price with which to send txns.
+   * @param {Object} param.votingContract DVM to query price requests.
+   * @param {Object} param.priceFeed Module used to query the current token price.
+   * @param {String} param.account Ethereum account from which to send txns.
+   * @param {Object} param.empProps Contains EMP contract state data. Expected:
    *      { crRatio: 1.5e18,
             minSponsorSize: 10e18,
             priceIdentifier: hex("ETH/BTC") }
-   * @param {Object} [config] Contains fields with which constructor will attempt to override defaults.
    */
-  constructor(logger, expiringMultiPartyClient, votingContract, gasEstimator, priceFeed, account, empProps, config) {
+  constructor({
+    logger = getLogger(),
+    expiringMultiPartyClient,
+    votingContract,
+    gasEstimator,
+    priceFeed,
+    account,
+    empProps,
+    config = LIQUIDATOR_CONFIG
```

Comment:
you fed in the config from the index as well as reading it in here from the `settings` file. is this not redundant?

Reply:
It was originally left out, but [due to how the current test suite was setup](https://github.com/UMAprotocol/protocol/blob/setup/liquidation-bot/liquidator/test/liquidator.js#L145). It was a easier to have it included as a default param for the constructor